### DISCUSSION TITLE
Support state class `total_increasing` for weight

### DISCUSSION THREAD

**Discussion by jbouwh** [home-assistant/architecture#1031] [Entity Models] [answered]
## Background

Some users use solid fuels, e.g. in case your energy consumption is based on the weight of pellets that are used. In that case it makes sense to see the amount of `kg` of pellets that are used over time in statistic sensors.

See:
https://github.com/home-assistant/core/issues/108436

## Proposal

Allow `SensorStateClass.TOTAL_INCREASING` to be used with `SensorDeviceClass.WEIGHT`.

**Comment by emontnemery** [comment, marked as answer]
The proposal was discussed and approved in a call January 25th

**Reply by jbouwh** [reply]
https://github.com/home-assistant/core/pull/108505 is opened to implement it.

**Reply by frenck** [reply]
Merged!
